inscriptions/ISic003379.xml

```diff
@@ -185,12 +185,15 @@
     <text>
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+                <ab>
+                    <lb n="1"/><expan><abbr>Sex</abbr><ex>tus</ex></expan> Al<hi rend="stacked">f</hi>iu<hi rend="small">s</hi>
```

Comment:
I would mark the f as small rather than stacked; the stacking is really a function of it's being small (it can sit above the tail of the L, as a result), rather than it having been inserted above the letter - it's identically treated as the final S, to all intents and purposes
https://theon.classics.ox.ac.uk/index.php/f/20365